cli/src/commands/auth.rs

```diff
@@ -0,0 +1,46 @@
+use crate::commands::Command;
+use crate::config::CliConfig;
+use crate::errors::{ExitCode, Fallible};
+use crate::telemetry::Session;
+use crate::terminal::{confirm, input};
+use log::{info, warn};
+use structopt::StructOpt;
+
+#[derive(StructOpt)]
+#[structopt(rename_all = "kebab-case")]
+pub enum Auth {
+    /// Setup your auth stuff
+    /// Requires using an User key which can be found here:
```

Comment:
Since we use `Personal API Key` in the UI, lets change this to:

```rs
/// Link the CLI to your Apollo account using your personal API key.
```

We can leave the url for the actual output since the `///` shows up in the `help` command